tests/phpunit/tests/theme/wpThemeJson.php

```diff
@@ -3874,4 +3874,19 @@ function data_set_spacing_sizes_when_invalid() {
 			),
 		);
 	}
+
+	/**
+	 * Tests that get_property_value static method returns an empty string
+	 * if the path is invalid.
+	 *
+	 * @ticket 56620
+	 */
+	public function test_get_property_value_should_return_string_with_invalid_path() {
+		$reflection_class = new ReflectionClass( WP_Theme_JSON::class );
+
+		$get_property_value_method = $reflection_class->getMethod( 'get_property_value' );
+		$get_property_value_method->setAccessible( true );
+		$result = $get_property_value_method->invoke( null, array(), array( 'non_existent_path' ) );
```

Comment:
Not strictly necessary, but good practice - reset the method to its original state after the invocation (and before the assertion):

```suggestion
		$get_property_value_method->setAccessible( true );
		$result = $get_property_value_method->invoke( null, array(), array( 'non_existent_path' ) );
		$get_property_value_method->setAccessible( false );
```

Reply:
Fixed in af7d40a682a9e7d6bd0406febf536aa90054cabb